docs/operations/hotplug_volumes.md

```diff
@@ -0,0 +1,124 @@
+This means that when you restart the VM, the disk is already defined in the VM, and thus in the new VMI.
+
+```bash
+$ virtctl addvolume vm-fedora --volume-name=example-volume-hotplug --persist
+```
+
+In the VM spec this will now show as a new disk
+```yaml
+    spec:
+    domain:
+        devices:
+            disks:
+            - disk:
+                bus: virtio
+                name: containerdisk
+            - disk:
+                bus: virtio
+                name: cloudinitdisk
+            - disk:
+                bus: scsi
+                name: example-volume-hotplug
+        machine:
+          type: ""
+```
+
+### Removevolume
+In addition to hotplug plugging the volume, you can also unplug it by using the 'removevolume' command available with virtctl
+```bash
+$ virtctl removevolume vmi-fedora --volume-name=example-volume-hotplug
+```
+**Note**: You can only unplug volumes that were dynamically added with addvolume, or using the API.
+
+### VolumeStatus
+VMI objects have a new VolumeStatus status field. This is an array containing each disk, hotplugged or not. For example after hotplugging the volume in the addvolume example, the VMI status will contain this:
+```yaml
+    volumeStatus:
+    - name: cloudinitdisk
+      target: vdb
+    - name: containerdisk
+      target: vda
+    - hotplugVolume:
+        attachPodName: hp-volume-7fmz4
+        attachPodUID: 62a7f6bf-474c-4e25-8db5-1db9725f0ed2
+      message: Successfully attach hotplugged volume volume-hotplug to VM
+      name: example-volume-hotplug
+      phase: Ready
+      reason: VolumeReady
+      target: sda
+```
+Vda is the container disk that contains the Fedora OS, vdb is the cloudinit disk. As you can see those just contain the name and target used when assigning them to the VM. The target is the value passed to QEMU when specifying the disks. The value is unique for the VM and does *NOT* represent the naming inside the guest. For instance for a Windows Guest OS the target has no meaning. The same will be true for hot plugged volumes. The target is just a unique identifier meant for QEMU, inside the guest the disk can be assigned a different name.
+
+The hotplugVolume has some extra information that regular volume statuses do not have. The attachPodName is the name of the pod that was used to attach the volume to the node the VMI is running on. If this pod is deleted it will also stop the VMI as we cannot guarantee the volume will remain attached to the node. The other fields are similar to conditions and indicate the status of the hot plug process. Once a Volume is ready it can be used by the VM.
+
+## Live Migration
+Currently Live Migration is enabled for any VMI that has volumes hotplugged into it. However there is a known issue that the migration may fail for VMIs with hotplugged block volumes if the target node uses CPU manager with static policy and `runc` prior to version `v1.0.0`.
```

Comment:
This `However` sentence should be an annotation.  Bring it down to a new line then use `!!! Warning` to engage the warning annotation.

Reply:
Done